### ISSUE TITLE
Add `Ignored Labels` to "Team settings" so certain PRs are removed from being assigned for reviews

### ISSUE THREAD

**Issue by annp89** [pullreminders/backlog#169]
In our automated release process, a PR is opened with the release branch that is deployed to our staging environments.

This PR has an `pullassigner-ignored` label automatically added to it. 

We would like the team to be excluded from being added to review this PR. Similar to `Ignored labels` in `My DM settings` can we get an option like so for the Team settings.

**Comment by abinoda** [comment]
@annp89 Hi Ann! Sorry, could you clarify a bit? Are you saying that PRs labeled with `pullassigner-ignored` currently are NOT getting properly ignored?

This is related to https://github.com/pullreminders/backlog/issues/142

**Comment by annp89** [comment]
Yes, even though our PRs are tagged with `pullassigner-ignored`, it still gets assigned to people on our team

**Comment by abinoda** [comment]
@annp89 Is it a bot that opens these release PRs? Is it a custom internal bot?

**Comment by annp89** [comment]
Yes it is
